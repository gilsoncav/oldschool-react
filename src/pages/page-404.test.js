import Page404 from './page-404';

describe('Page404 component', () => {
  it('renders correctly Page404', () => {
    const wrapper = shallow(<Page404 />);
    expect(wrapper).toMatchSnapshot();
  });
});
