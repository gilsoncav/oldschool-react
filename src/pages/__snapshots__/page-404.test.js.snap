// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404 component renders correctly Page404 1`] = `
<div>
  <Row
    gutter={0}
  >
    <Col
      offset={6}
      span={12}
      style={
        Object {
          "paddingTop": "10vh",
        }
      }
    >
      <div
        className="clean-card text-center"
      >
        <h1
          className="text-primary text-baf"
          style={
            Object {
              "marginBottom": "0",
            }
          }
        >
          404
        </h1>
        <h1
          className="text-primary"
          style={
            Object {
              "marginBottom": "0",
            }
          }
        >
          It seems to me that you're lost!
        </h1>
        <Link
          to="/"
        >
          <Button
            block={false}
            className="btn-secondary"
            ghost={false}
            htmlType="button"
            icon="rollback"
            loading={false}
            size="large"
            style={
              Object {
                "marginBottom": "15px",
              }
            }
          >
            Take Me Back Home
          </Button>
        </Link>
      </div>
    </Col>
  </Row>
</div>
`;
